ci-operator/step-registry/mirror-images/by-oc-mirror/mirror-images-by-oc-mirror-commands.sh

```diff
@@ -98,53 +91,48 @@ extract_oc_mirror "${new_pull_secret}" "${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRI
 oc_mirror_bin="${oc_mirror_dir}/oc-mirror"
 run_command "'${oc_mirror_bin}' version --output=yaml"
 
-# pre-check the expected image exists in official channel
-if "${oc_mirror_bin}" list releases --filter-by-archs ${ocp_image_arch} --channel=${channel_name} | grep -q "^${readable_version}$"; then
-    echo "${readable_version} is found in ${channel_name} on ${ocp_image_arch} arch"
-else
-    echo "${readable_version} is NOT found in ${channel_name} on ${ocp_image_arch} arch, exit..."
-    exit 1
-fi
 
 # set the imagesetconfigure
 image_set_config="image_set_config.yaml"
 cat <<END | tee "${image_set_config}"
 kind: ImageSetConfiguration
-apiVersion: mirror.openshift.io/v1alpha2
+apiVersion: mirror.openshift.io/v2alpha1
 mirror:
   platform:
-    architectures:
-      - "${ocp_image_arch}"
-    channels:
-      - name: ${channel_name}
-        minVersion: ${readable_version}
-        maxVersion: ${readable_version}
+    release: ${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE}
 END
 
 # https://github.com/openshift/oc-mirror/blob/main/docs/usage.md#authentication
 # oc-mirror only respect ~/.docker/config.json -> ${XDG_RUNTIME_DIR}/containers/auth.json
 mkdir -p "${XDG_RUNTIME_DIR}/containers/"
 cp -rf "${new_pull_secret}" "${XDG_RUNTIME_DIR}/containers/auth.json"
 
+unset REGISTRY_AUTH_PREFERENCE
+
 # execute the oc-mirror command
-cmd="'${oc_mirror_bin}' -c ${image_set_config} docker://${target_release_image_repo} --dest-skip-tls"
+cmd="'${oc_mirror_bin}' -c ${image_set_config} docker://${target_release_image_repo} --dest-tls-verify=false --v2 --workspace file://${oc_mirror_dir}"
 echo "Command: $cmd"
 eval "$cmd" | tee "${mirror_output}"
```

Comment:
If we are not capturing generated mirror config file info from oc-mirror command output, then we can directly run the following command to replace the above 3 lines:
```
run_command "'${oc_mirror_bin}' -c ${image_set_config} docker://${target_release_image_repo} --dest-tls-verify=false --v2 --workspace file://${oc_mirror_dir}"
```